src/App.jsx

```diff
@@ -37,21 +41,21 @@ export function App() {
   var instructions = appInfo.instructions.start
   
   var currentMenuItems = [];
-  if (categ !== '' && res === ''){
+  if (isStringEmpty(categ) && isStringEmtpy(res)) {
```

Comment:
Negate appropriate isStringEmpty calls. 
```suggestion
  if (!isStringEmpty(categ) && isStringEmtpy(res)) {
```